csfieldguide/chapters/content/en/coding-compression/sections/general-purpose.md

```diff
@@ -13,7 +13,7 @@ The interactive below illustrates this idea using a variation of the Ziv-Lempel
 
 In LZSS, a reference to where some text occurred before is actually two numbers: the first says how many characters to count back to where the previous phrase starts, and the second says how long the referenced phrase is.
 Each reference typically takes the space of about one or two characters, so the system makes a saving as long as two characters are replaced.
-The options in the interactive above allow you to require the replaced length to be at least two characters, to avoid replacing a single character with a reference.
+The interactive above requires the replaced length to be at least two characters, this avoids replacing a single character with a reference.
```

Comment:
Actually do we need this sentence at all? It's self explanatory and the sentence is now a repeat of the one previous